pkg/controller/jobs/kubeflow/jobs/mxjob/mxjob_controller_test.go

```diff
@@ -22,7 +22,6 @@ import (
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
 	kftraining "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
-	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
```

Comment:
Actually, remove this one.

Most of the code is written with `corev1`.

Reply:
Changed.